yb-voyager/src/query/queryissue/detectors_ddl.go

```diff
@@ -533,6 +540,16 @@ func (tid *TriggerIssueDetector) DetectIssues(obj queryparser.DDLObject) ([]Quer
 		))
 	}
 
+	if slices.Contains(unsupportedLargeObjectFunctions, trigger.FuncName) {
```

Comment:
@sanyamsinghal can you review this? Maybe we can enhance FuncCall detector?

Reply:
I don't think we do much for this in FuncCallDetector, I checked its parsed tree its a plain `pg_query.Node` type of field in create trigger stmt so can't differentiate that from a func call or anything.

Reply:
Got it.